lib/attributor/attribute.rb

```diff
@@ -137,8 +137,7 @@ def example_from_options(parent, context)
       when ::Regexp
         val.gen
       when ::Array
-        # TODO: handle arrays of non native types, i.e. arrays of regexps.... ?
-        val.pick
+        val
```

Comment:
Or we can just remove the special handling for arrays entirely. In which case, simply kull this whole `when ::Array` section entirely.